descholar-ceo/AnnounceIT
# #170787286 Administrator sees his admin page

#### What does this PR ​ do​ ?
As an admin, I should be able to see my admin page

#### Description of Task to be completed?
* After an admin logs in, he/she will be able to see his/her own admin-page
* Admin clicks on logout button in the navbar, and he is redirected to the signin page
* Admin clicks on Create and he/she get redirected to create new announcement page

#### How should this be manually tested?
* Clone this repository in case you don't have it using this command `git clone https://github.com/descholar-ceo/AnnounceIT.git`
* If you use [VSCode](https://code.visualstudio.com/) as code editor, install [_live-server_](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer), and click on _Go Live_

* Click on enter button from homepage, 

* And then you will be redirected in the browser, here you will need to go to the root of the url you have in url-bar, and enter like `http://127.0.0.1:5500/UI/html/admin-side/admin-page.html`, will be redirected to the admin dashboard, or enter like this `http://127.0.0.1:5500/UI/html/admin-side/admin-page.html`

* Or if you don't use [VSCode](https://code.visualstudio.com/) as text editor, after cloning the repo, you navigate in `AnnounceIT/UI/html/admin-side/` and you will see different files, you choose `admin-page.html` and right click and open with your preferred browser

#### Any background context you want to provide?
* It would be better if you use  [VSCode](https://code.visualstudio.com/)

#### What are the relevant pivotal tracker stories?
[170787286](https://www.pivotaltracker.com/n/projects/2429498#)

#### Screenshots (​ if​ appropriate)
![Screenshot from 2020-01-19 13-10-28](https://user-images.githubusercontent.com/56886064/72679939-198b2200-3abd-11ea-9193-b93a098260b4.png)

#### Questions:
No Question
